### PR TITLE
[CI] Migrate all Jenkins jobs in bldenv folder

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
Migrate 5 Jenkins jobs in bldenv folder.
1. dpkg-cache-clean 
2. image-building for sonic-mgmt
3. image-building for sonic-slave-[buster|stretch|jessie].
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: .azure-pipelines/bldenv/docker-sonic-mgmt.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      submodules: recursive
+    - script: |
+        set -xe
+        make configure PLATFORM=generic
+        make target/docker-sonic-mgmt.gz
+      displayName: Build docker-sonic-mgmt.gz
+    - script: |
+        set -xe
+        ID=`id -un`
+        sudo mkdir /home/$ID/.docker -p
```
Comment (liushilongbuaa) [inline .azure-pipelines/bldenv/docker-sonic-mgmt.yml:38]: command 'docker login' need this directory.

File: .azure-pipelines/scripts/docker-upload.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
```
Comment (xumia) [inline .azure-pipelines/scripts/docker-upload.sh:1]: Maybe We can change to use azure pipelines task instead.
https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/containers/push-image?view=azure-devops

- task: Docker@2
  displayName: Push image
  inputs:
    containerRegistry: |
      $(dockerHub)
    repository: $(imageName)
    command: push
    tags: |
      test1
      test2

File: .azure-pipelines/bldenv/dpkg-cache-cleanup.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight run
+  branches:
+    include:
+    - master
```
Comment (xumia) [inline .azure-pipelines/bldenv/dpkg-cache-cleanup.yml:11]: Missing always run?

File: .azure-pipelines/bldenv/dpkg-cache-cleanup.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Starter pipeline
```
Comment (xumia) [inline .azure-pipelines/bldenv/dpkg-cache-cleanup.yml:1]: Can we check in the azure pipeline into another repo?

**Pull request closed without merging**